app/adapters/application.js

```diff
@@ -35,6 +35,12 @@ export default HalAdapter.extend({
     } else {
       return error;
     }
+  },
+
+  deleteRecord: function(store, type, record){
```

Comment:
@bantic I think this should be all that is needed to force deletes to hit the non-nested route. I just don't pass the record into `buildURL`, so there is no nesting.